<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowReturnView</class>
 <widget class="QWidget" name="BorrowReturnView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leReaderId">
       <property name="placeholderText">
        <string>请输入读者id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leBookId">
       <property name="placeholderText">
        <string>请输入图书id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBorrow">
       <property name="text">
        <string>借书</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReturn">
       <property name="text">
        <string>还书</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
